src/main/java/org/quiltmc/loader/impl/plugin/QuiltPluginManagerImpl.java

```diff
@@ -1585,6 +1585,12 @@ public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs) th
 							return FileVisitResult.SKIP_SUBTREE;
 						}
```

Comment:
This can be removed now

Reply:
No? This check only occurs at the *end* of the folder name - although they could probably be merged instead.

Reply:
Oops, yes, my mind misread "ends" as "starts" for whatever reason